packages/create-svelte/templates/skeletonlib/package.template.json

```
@@ -3,8 +3,15 @@
 	"version": "0.0.1",
 	"scripts": {
 		"dev": "vite dev",
-		"build": "svelte-kit sync && svelte-package",
-		"prepublishOnly": "echo 'Did you mean to publish `./package/`, instead of `./`?' && exit 1"
+		"build": "vite build && npm run package",
+		"package": "svelte-kit sync && svelte-package"
+	},
+	"exports": {
+		".": "./dist/index.js",
+		"./package.json": "./package.json"
```

IIRC the main reason we encourage people to expose `package.json` was so that tooling could read `pkg.svelte` easily, but I think that's unnecessary if we use the `svelte` condition in `exports`?

```suggestion
		".": {
		  "types": "./dist/index.d.ts",
		  "svelte": "./dist/index.js",
		  "default": "./dist/index.js"
		}
```

---

AFAIK roll-up plugin svelte to this day throws a warning (or even error?) if it detects a Svelte package that doesn't do this. Either way, earlier versions did so for backwards compatibility it would be good to have it.

---

Update: Checked this, indeed `rollup-plugin-svelte` to this day needs the `package.json` export. We should fix that.

---

Update 2: https://github.com/sveltejs/rollup-plugin-svelte/pull/198 solves this. Someone with access should merge and release this (patch release is best so as many people as possible get the update).

---

hmm. i merged it, it does not solve it. not sure how CI passed on that PR, it's got problems

---

ah, it's because there is no CI

---

dammit, i should not have merged that PR 🙄

---

opened a new one: https://github.com/sveltejs/rollup-plugin-svelte/pull/205